jerry-core/ecma/base/ecma-globals.h

```diff
@@ -204,6 +204,16 @@ typedef struct
   void *info_p; /**< free info or callback */
 } ecma_native_pointer_t;
 
+#ifdef JERRY_VM_EXEC_STOP
+
+/**
+ * Callback which returns with a non-undefined value
+ * when the ECMAScript execution should be stopped.
+ */
+typedef ecma_value_t (*jerry_vm_exec_stop_cb_t) (void *user_p);
```

Comment:
This is very bad news, I am not aware of that. I don't think removing this dependency should be in this patch though.